common-modules/common-service/src/main/java/com/google/cloud/healthcare/fdamystudies/common/ErrorCode.java

```diff
@@ -162,7 +162,7 @@
       400,
       "EC_0032",
       Constants.BAD_REQUEST,
-      "User should have at least one permission in the system to access this resource."),
+      "The admin should have atleast one permission in the system to access this resource"),
```

Comment:
Typos - please be careful...

Reply:
Fixed review comment

Reply:
atleast is not a word

Reply:
I have updated this typo error [ErrorCode](https://github.com/GoogleCloudPlatform/fda-mystudies/blob/2.0.2-release-candidate/common-modules/common-service/src/main/java/com/google/cloud/healthcare/fdamystudies/common/ErrorCode.java)